pkg/migrations/op_add_column_test.go

```diff
@@ -674,7 +766,7 @@ func TestAddColumnValidation(t *testing.T) {
 					},
 				},
 			},
-			wantStartErr: migrations.InvalidPrimaryKeyError{Table: "orders", Fields: 2},
+			wantStartErr: migrations.BackfillNotPossible{Table: "orders"},
```

Comment:
I think another test case is needed here to test that an operation that requires backfill is valid if it doesn't have a PK but does have a unique, not null column.

Reply:
I added `add column with up sql and no pk` for this

Reply:
ah sorry, got it now, adding it 🙏

Reply:
pushed [a28aec0](https://github.com/xataio/pgroll/pull/288/commits/a28aec0fc4e8a95981f710d2be00c50ad9aa27d2)